build/IPDK_Container/scripts/host_install.sh

```diff
@@ -154,8 +154,8 @@ else
 fi
 
 cd /root || exit
-cp -r /git/ipdk/scripts scripts
-cp -r /git/ipdk/examples examples
+cp -r /git/ipdk/scripts /root
+cp -r /git/ipdk/examples /root
```

Comment:
Why don't use the dot (.) As two lines above there is a statement going into the /root directory....

Reply:
@mestery there is no pushd only cd!!

Reply:
Fixed in #55, it should have been pushd.